q2_quality_control/decontam.py

```diff
@@ -58,27 +71,23 @@ def _check_column_inputs(table, metadata, method, freq_concentration_column,
                          'metadata:\n'
                          + str(', '.join(no_info)))
     if method == 'prevalence':
-        _check_column_inputs_helper(table, metadata,
-                                    prev_control_column,
-                                    prev_control_indicator)
-
+        if freq_concentration_column is not None:
+            print("Warning: --p-freq-concentration-column given, but" +
```

Comment:
You'll want to use the `warnings` module for warnings instead of print so that the warnings status as a warning is clear (and presented to a user in all cases). `print` sends to a different output stream and is ignored in qiime2 without the `--verbose` flag. 

If you don't want to worry about all that it could make sense to simply error on unexpected inputs so that it's made very explicit to users that the parameter is not being taken into account.

Reply:
I opted for just raising a value error instead of the warning as to avoid confusion with users in the future about what parameters are being utilized. Addressed in commit [f31210b](https://github.com/qiime2/q2-quality-control/pull/88/commits/f31210b1b4524e53c8545435910ecb106d40fc8d)